basics/node/std_node.md

```diff
@@ -0,0 +1,125 @@
+# The `std/node` Library
+
+The `std/node` part of the Deno standard library is a Node compatibility layer.
+Its primary focus is providing "polyfills" for Node's
+[built-in modules](https://github.com/denoland/deno_std/tree/main/node#supported-builtins).
+It also provides a mechanism for loading CommonJS modules into Deno.
+
+The library is most useful when trying to use your own or private code that was
+written for Node. If you are trying to consume public npm packages, you are
+likely to get a better result using a [CDN](./cdns.md).
+
+## Node built-in modules
+
+The standard library provides several "replacement" modules for Node built-in
+modules. These either replicate the functionality of the built-in or they call
+the Deno native APIs, returning an interface that is compatible with Node.
+
+The standard library provides modules for the following built-ins:
+
+- [`assert`](https://deno.land/std/node/assert.ts) (_partly_)
```

Comment:
@kt3k @cjihrig is this list current and accurate?

Reply:
Sorry, I missed this ping. I think a few things are missing. There is [this list](https://github.com/denoland/deno_std/tree/main/node#supported-modules), which I think is more up to date, but is also missing things like the recently added `readline/promises` module.

Reply:
ok I updated the docs to reflect the list